doc/man3/OPENSSL_LH_COMPFUNC.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LHASH, DECLARE_LHASH_OF,
```

Comment:
I think the DECLARE_LHASH_OF was meant to be just LHASH_OF which is mentioned in the text.

Reply:
Also the DECLARE_LHASH_HASH_FN and DECLARE_LHASH_COMP_FN seem to be documented too already.

Reply:
Also there is an incorrect link to DECLARE_LHASH_OF from migration_guide(7) manpage.

Reply:
Not sure I follow you. I already updated `migration_guide(7)` in the last commit. I can list `LHASH_OF` in the synopsis.

Reply:
> I already updated `migration_guide(7)` in the last commit.

Ah, I did not notice somehow.

Yes, list also LHASH_OF which is already documented in the manpage text.

Reply:
Done.